docs/content/connectors/mongodb-cdc.md

```diff
@@ -506,3 +506,9 @@ Reference
 - [Connection String Options](https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-connection-options)
 - [BSON Types](https://docs.mongodb.com/manual/reference/bson-types/)
 - [Flink DataTypes](https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/table/types/)
+
+[FAQ](https://github.com/ververica/flink-cdc-connectors/wiki/FAQ)
+--------
+
+[FAQ(ZH)](https://github.com/ververica/flink-cdc-connectors/wiki/FAQ(ZH))
+--------
```

Comment:
Thanks for the contribution @rookiegao . How bout changing the style to following?
```
FAQ
--------
* [FAQ(English)](https://github.com/ververica/flink-cdc-connectors/wiki/FAQ)
* [FAQ(中文)](https://github.com/ververica/flink-cdc-connectors/wiki/FAQ(ZH))
```